### PR TITLE
Fix single track PlanB offers

### PR DESCRIPTION
**Pull request by Sean-Der**
We had an off-by-one when generating PlanB offers. Fix and add a test

### CODE REVIEW

File: peerconnection_media_test.go [hunk 1]
```diff
@@ -1036,63 +1036,75 @@ func TestGetRegisteredRTPCodecs(t *testing.T) {
...
-	assert.NoError(t, pcOffer.Close())
-	assert.NoError(t, pcAnswer.Close())
+	report := test.CheckRoutines(t)
+	defer report()
+
+	t.Run("Single Track", func(t *testing.T) {
+		runTest(1)
+	})
+	t.Run("Multi Track", func(t *testing.T) {
+		runTest(2)
```
Comment (jeremija) [inline peerconnection_media_test.go:1106]: I'd pass `t` as an argument to `runTest` to ensure we get the details about which test case failed in the test logs.

**Comment by jeremija** [comment]
Hmm weird test failures in the logs. Has this been common recently?

**Comment by at-wat** [comment]
Timeout error appeared on the master branch recently.
https://travis-ci.org/github/pion/webrtc/jobs/693247808

Another error is
```
--- FAIL: TestDataChannelParamters_Go (20.41s)
    --- FAIL: TestDataChannelParamters_Go/MaxPacketLifeTime_exchange (10.01s)
        datachannel_test.go:36: closePair timed out waiting for done signal
```

File: peerconnection_media_test.go (hunk 1 above)
Comment (Sean-Der) [reply peerconnection_media_test.go:1106]: Fixed!

**Comment by Sean-Der** [comment]
@at-wat @jeremija yea we have had issues with that test in the past :/

Let me look into how I can get it to dump a stack trace on failure, would help a lot more in figuring out where it is deadlocking.

File: peerconnection_media_test.go (hunk 1 above)
Comment (Sean-Der) [reply peerconnection_media_test.go]: Fixed!

**Comment by Sean-Der** [comment]
Thanks for the quick review @jeremija! Mind approving the PR so I can merge :)

**Comment by jeremija** [comment]
Sorry it always takes me so long to find the approve button on my phone haha

**Comment by Sean-Der** [comment]
lol just because we know how to program, doesn't mean we know how to use computers ;)

**Pull request merged**